Source/WebCore/page/scrolling/mac/ScrollerMac.mm

```diff
@@ -380,8 +380,14 @@ - (void)invalidate
 
 void ScrollerMac::updateScrollbarStyle()
 {
-    setScrollerImp([NSScrollerImp scrollerImpWithStyle:nsScrollerStyle(m_pair.scrollbarStyle()) controlSize:nsControlSizeFromScrollbarWidth(m_pair.scrollbarWidthStyle()) horizontal:m_orientation == ScrollbarOrientation::Horizontal replacingScrollerImp:takeScrollerImp().get()]);
+    takeScrollerImp();
```

Comment:
I don't think the `take` here is necessary; ` setScrollerImp` does an assignment to `m_scrollerImp` so nulling first shouldn't make a difference.

Reply:
Fixed.